### DISCUSSION TITLE
groupadd: invalid group ID 'sail' on Laravel 8 in Docker Desktop Windows

### DISCUSSION THREAD

**Discussion by JimmyTzuc** [microsoft/WSL#6667] [Q&A]
# ERROR

```
Service 'laravel.test' failed to build : The command '/bin/sh -c groupadd --force -g $WWWGROUP sail' returned a non-zero code: 3
```

# Steps to reproduce

Hi have a problem when I execute ```docker compose up -d```  or  ```./vendor/bin/sail up -d``` in my laravel project with Docker Desktop on Windows

**Comment by gavinservai** [comment]
Hey just checking in, were you able to find a fix or work-around for this?

**Reply by Oxicode** [reply]
:(

**Comment by jessinthecloud** [comment]
In case it helps anyone that finds this later, putting this in my `.env` fixed the `groupadd: invalid group ID 'sail'` problem for me:
```
WWWGROUP=1000
WWWUSER=1000
```

From this stackoverflow answer: https://stackoverflow.com/a/68244277
